pkg/trace/trace.go

```diff
@@ -16,13 +16,21 @@ package trace
 
 import (
 	"fmt"
+	"os"
 	"runtime"
 	"time"
 
 	"github.com/Sirupsen/logrus"
 )
 
-var Logger = logrus.New()
+var Logger = &logrus.Logger{
+	Out: os.Stderr,
+	// We're using our own text formatter to skip the \n and \t escaping logrus
+	// was doing on non TTY Out (we redirect to a file) descriptors.
+	Formatter: &TextFormatter{},
+	Hooks:     make(logrus.LevelHooks),
+	Level:     logrus.InfoLevel,
```

Comment:
Do we need https://github.com/vmware/vic/pull/2751/files#diff-6b636c14743b03d96931844d09206b91R32?